WISE-Community/WISE
# Changed project terminology in the wise5 apps

To test:
- Open Authoring Tool and make sure that all instances of 'project' have been changed to 'unit' and 'activity' has been changed to 'lesson'.
- Do the same for the Classroom Monitor.

Resolves #2224.
